src/main/java/com/sap/sgs/phosphor/fosstars/data/github/UsesOwaspDependencyScan.java

```diff
@@ -0,0 +1,292 @@
+   */
+  private static boolean isDependencyCheck(Plugin plugin) {
+    return isDependencyCheck(plugin.getGroupId(), plugin.getArtifactId());
+  }
+
+  /**
+   * Check if a report plugin is OWASP Dependency Check plugin.
+   *
+   * @param plugin The report plugin to be checked.
+   * @return True if the report plugin is OWASP Dependency Check plugin, false otherwise.
+   */
+  private static boolean isDependencyCheck(ReportPlugin plugin) {
+    return isDependencyCheck(plugin.getGroupId(), plugin.getArtifactId());
+  }
+
+  /**
+   * Check if groupId and artifactId matches with OWASP Dependency GA.
+   *
+   * @param groupId to be checked.
+   * @param artifactId to be checked.
+   * @return True if the OWASP Dependency GA matches, false otherwise.
+   */
+  private static boolean isDependencyCheck(String groupId, String artifactId) {
+    return "org.owasp".equals(groupId) && "dependency-check-maven".equals(artifactId);
+  }
+
+  /**
+   * Check the usage of OWASP Dependency Check plugin in the project.
+   * 
+   * @param locations Set of location hits which indicates where exactly the plugin was found.
+   * @return The usage status of type {@link OwaspDependencyCheckUsage}.
+   */
+  private static OwaspDependencyCheckUsage usageFrom(Set<Location> locations) {
+    if (locations == null || locations.isEmpty()) {
+      return NOT_USED;
+    }
+
+    if (mandatoryCheckFor(locations)) {
+      return MANDATORY;
+    }
+
+    return OPTIONAL;
+  }
+
+  /**
+   * Check if plugin configurations have an attribute which can fail the build on finding
+   * vulnerabilities above a given CVSS score. Hence, look for possible CVSS score value which
+   * indicates, when to fail the build.
+   *
+   * @param configuration The plugin configuration to be checked.
+   * @return The cvss score which indicates how the build should fail. Otherwise null.
+   */
+  private static Double cvssScoreFrom(Object configuration) {
+    // The OWASP Dependency Check plugin is found and checking configurations.
+    for (String name : FAIL_BUILD_CONFIGURATIONS.keySet()) {
+      Optional<String> content = parameter(name, configuration);
+
+      if (content.isPresent()) {
+        String type = FAIL_BUILD_CONFIGURATIONS.get(name);
+        return parseScore(content.get(), type);
+      }
+    }
+
+    return null;
+  }
+
+  /**
+   * Parse the value depending on the given type. The type can be 'boolean' or 'number'. If
+   * 'boolean', then return 0.0, otherwise parse the given string value to 'number'. If the input is
+   * null or not parseable, then an exception is thrown.
+   * 
+   * @param value the value to be parsed.
+   * @param type the type of the string value.
+   * @return the parsed value.
+   */
+  private static double parseScore(String value, String type) {
+    switch (type) {
+      case "number":
+        return Double.parseDouble(value);
+      case "boolean":
+        return 0.0;
```

Comment:
If `failBuildOnAnyVulnerability` is explicitly set to `false`, should the method return `0.0`?

Reply:
Yes I think it should. If `0.0`, then any CVSS score should be considered, right? 
Or is there something else I may be missing?

Reply:
I think if `failBuildOnAnyVulnerability` is false, then the threshold is not specified (unless it's set in another option).